plugins/lando-services/services/mariadb/builder.js

```diff
@@ -8,13 +8,13 @@ module.exports = {
   name: 'mariadb',
   config: {
     version: '10.3',
-    supported: ['10.5', '10.4', '10.3', '10.2', '10.1'],
+    supported: ['10.6', '10.5', '10.4', '10.3', '10.2'],
     pinPairs: {
-      '10.5': 'bitnami/mariadb:10.5.8-debian-10-r74',
-      '10.4': 'bitnami/mariadb:10.4.17-debian-10-r84',
-      '10.3': 'bitnami/mariadb:10.3.27-debian-10-r84',
-      '10.2': 'bitnami/mariadb:10.2.36-debian-10-r83',
-      '10.1': 'bitnami/mariadb:10.1.47-debian-10-r13',
```

Comment:
Maintaining the pin pairs helps ensure compatibility...it could be good to update the pinPairs if you can show the tests all passing and with some manual testing to ensure that updates don't introduce regressions.

Reply:
Good point, I'll have them restored right away, if there's anything I can do with the manual testing just let me know.